include/rime_api.h

```diff
@@ -101,6 +101,7 @@ typedef struct {
   int num_candidates;
   RimeCandidate* candidates;
   char* select_keys;
+  char** select_labels;
```

Comment:
Here is a subtle issue. RimeContext is an expandable "RIME_STRUCT" but RimeMenu isn't. If we add new fields here we'll break binary compatibility with dynamically linked client apps: an old client app may create a RimeContext without this field and pass it to librime.
We'd better add new fields at the end of RimeContext, then we'll be able to check RIME_STRUCT_HAS_MEMBER(*context, context->new_field) and avoid accessing past the structs' size.